src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/SearchController.cs

```diff
@@ -142,6 +135,28 @@ private List<SearchResult> FuzzySearchOpenWindows(List<Window> openWindows)
             return result;
         }
 
+        /// <summary>
+        /// Search method that matches all the windows with a title
+        /// </summary>
+        /// <param name="openWindows">what windows are open</param>
+        /// <returns>Returns search results</returns>
+        private List<SearchResult> AllOpenWindows(List<Window> openWindows)
+        {
+            List<SearchResult> result = new List<SearchResult>();
+
+            foreach (var window in openWindows)
+            {
+                if (window.Title.Length != 0)
+                {
+                    result.Add(new SearchResult(window));
+                }
+            }
+
+            System.Diagnostics.Debug.Print("Found " + result.Count + " windows that match the search text");
+
+            return result.OrderBy(w => w.Result.Title).ToList();
```

Comment:
Why not order by Z-Index. We can have a setting to switch between up or descending index order. (Can be a second step improvement.)

Reply:
I am fine with alphabetical and I am not sure if we have the z-index or if require another interop call.

Reply:
We don't have it. It requires interop calls and enumerations. I am also not sure if it will slow down things.